molecules/_dropdown.scss

```diff
@@ -110,7 +110,7 @@ $dropdown-background-color-accent: $earl-gray !default;
     display: block;
     .dropdown-container {
       @include dropdown-container($dropdown-width, $dropdown-background-color, $dropdown-shadow);
-      &.wide {
+      &.mod-wide {
```

Comment:
what is the mod for ? some sort of naming ? If yes, we should document it somewhere, e.g. COntributing.md or Wiki, Readme